src/core/lib/transition-manager.js

```diff
@@ -20,7 +17,7 @@ export const TRANSITION_EVENTS = {
 const DEFAULT_PROPS = {
   transitionDuration: 0,
   transitionEasing: t => t,
-  transitionInterpolator: viewportLinearInterpolator,
+  transitionInterpolator: new ViewportLinearInterpolator(),
```

Comment:
Thought: While fine here, could this API cause a new Interpolator to be created on every render in apps when they use this?
```
<DeckGL
   transitionInterpolator={new ViewportLinearInterpolator()}
>
```
Are interpolator instances guaranteed to be stateless so this always works?